linkerd/app/outbound/src/tests.rs

```diff
@@ -1,51 +1,35 @@
 use crate::Config;
-use indexmap::indexset;
-use linkerd2_app_core::{self as app_core, metrics::Metrics, Addr};
+use futures::prelude::*;
+use linkerd2_app_core::{config, exp_backoff, proxy::http::h2, transport::listen, Addr, Error};
 use linkerd2_app_test as test_support;
 use std::{net::SocketAddr, time::Duration};
 use tower::ServiceExt;
 
 const LOCALHOST: [u8; 4] = [127, 0, 0, 1];
-const LISTEN_PORT: u16 = 4140;
 
 fn default_config(orig_dst: SocketAddr) -> Config {
-    use app_core::{
-        config::{ConnectConfig, ProxyConfig, ServerConfig},
-        exp_backoff::ExponentialBackoff,
-        proxy::http::h2,
-        transport::listen,
-    };
-    let h2_settings = h2::Settings {
-        initial_stream_window_size: Some(65_535), // Protocol default
-        initial_connection_window_size: Some(1_048_576), // 1MB ~ 16 streams at capacity
-    };
     Config {
         canonicalize_timeout: Duration::from_millis(100),
-        proxy: ProxyConfig {
-            server: ServerConfig {
-                bind: listen::Bind::new(SocketAddr::new(LOCALHOST.into(), LISTEN_PORT), None)
+        proxy: config::ProxyConfig {
+            server: config::ServerConfig {
+                bind: listen::Bind::new(SocketAddr::new(LOCALHOST.into(), 0), None)
                     .with_orig_dst_addr(orig_dst.into()),
-                h2_settings,
+                h2_settings: h2::Settings::default(),
```

Comment:
i think this `Default` impl returns different settings than the actual default values defined in `env.rs`; IIRC it's derived and these are both `None`.

Reply:
yeah, it's not a big deal, i just thought it was better for this to match the actual defaults everywhere, so that we don't go to write a test that _does_ depend on some setting and get surprising results. not a blocker though.